Author: Alastair McKinstry <mckinstry@debian.org>
Description: Disable downloading grib files; already present
Last-Updated: 2019-07-15
Forwarded: not-needed

--- a/cmake/ecbuild_get_test_data.cmake
+++ b/cmake/ecbuild_get_test_data.cmake
@@ -10,7 +10,7 @@
 
 # function for downloading test data
 
-function( _download_test_data _p_NAME _p_DIR_URL _p_DIRLOCAL _p_CHECK_FILE_EXISTS )
+ function( _download_test_data _p_NAME _p_DIR_URL _p_DIRLOCAL _p_CHECK_FILE_EXISTS )
 
   # TODO: make that 'at ecmwf'
   #if(1)
@@ -224,7 +224,7 @@ function( ecbuild_get_test_data )
 
     # download the data
 
-    _download_test_data( ${_p_NAME} ${DOWNLOAD_URL} ${_p_DIRLOCAL} ${CHECK_FILE_EXISTS} )
+    # _download_test_data( ${_p_NAME} ${DOWNLOAD_URL} ${_p_DIRLOCAL} ${CHECK_FILE_EXISTS} )
 
     # perform the checksum if requested
 
@@ -289,13 +289,13 @@ function( ecbuild_get_test_data )
 
     endif()
 
-    add_custom_target( ${_p_TARGET} DEPENDS ${_deps} )
+#    add_custom_target( ${_p_TARGET} DEPENDS ${_deps} )
 
-    if( _p_EXTRACT )
-      ecbuild_debug("ecbuild_get_test_data: extracting ${_p_DIRLOCAL}/${_p_NAME} (post-build for target ${_p_TARGET}")
-      add_custom_command( TARGET ${_p_TARGET} POST_BUILD
-                          COMMAND ${CMAKE_COMMAND} -E chdir ${_p_DIRLOCAL} tar xvf ${_p_NAME} )
-    endif()
+#    if( _p_EXTRACT )
+#      ecbuild_debug("ecbuild_get_test_data: extracting ${_p_DIRLOCAL}/${_p_NAME} (post-build for target ${_p_TARGET}")
+#      add_custom_command( TARGET ${_p_TARGET} POST_BUILD
+#                         COMMAND ${CMAKE_COMMAND} -E chdir ${_p_DIRLOCAL} tar xvf ${_p_NAME} )
+#    endif()
 
 endfunction(ecbuild_get_test_data)
 
