--- a/cmake/ecbuild_check_os.cmake
+++ b/cmake/ecbuild_check_os.cmake
@@ -266,7 +266,7 @@
       # If this is the case for the current linker, disable checking for dynamic symbols
       # See https://sourceware.org/bugzilla/show_bug.cgi?id=20535
 
-      set( _linker_check_srcdir "${ECBUILD_MACROS_DIR}/../check_linker" )
+      set( _linker_check_srcdir "/usr/share/ecbuild/check_linker" )
       if( NOT EXISTS ${_linker_check_srcdir} ) # ecbuild source dir
         set( _linker_check_srcdir "${ECBUILD_MACROS_DIR}/../share/ecbuild/check_linker" )
       endif()
--- a/cmake/ecbuild_install_project.cmake
+++ b/cmake/ecbuild_install_project.cmake
@@ -202,7 +202,7 @@
     # Find the ecbuild linker check files and include in the source package if found
     find_path( ECBUILD_LINKCHECK_DIR dso1.c
                PATHS ${ECBUILD_MACROS_DIR}/../check_linker
-                     ${ECBUILD_MACROS_DIR}/../share/ecbuild/check_linker )
+                     /usr/share/ecbuild/check_linker )
 
     mark_as_advanced( ECBUILD_LINKCHECK_DIR )
 
