--- a/memfs/CMakeLists.txt
+++ b/memfs/CMakeLists.txt
@@ -48,7 +48,15 @@
 endif()
 
 if( HAVE_MEMFS )
-    ecbuild_add_library(TARGET    eccodes_memfs
+  ecbuild_add_library(TARGET    eccodes_memfs
+    VERSION 0
+    CFLAGS ${CFLAGS}
+                        SOURCES   ${generated_c_files}
+                        GENERATED ${generated_c_files})
+  ecbuild_add_library(TARGET    eccodes_memfs_static
+    VERSION 0
+    CFLAGS ${CFLAGS}
+    TYPE static 
                         SOURCES   ${generated_c_files}
                         GENERATED ${generated_c_files})
 endif()
