Author: Alastair McKinstry <mckinstry@debian.org>
Description: Make builds bit-reproducible
Last-Updated: 2016-10-27
Forwarded: https://software.ecmwf.int/issues/browse/SUP-1590 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -146,6 +146,12 @@
     DESCRIPTION "Build the ecCodes Fortran interface"
     DEFAULT ON )
 
+ecbuild_add_option( FEATURE BIT_REPRODUCIBLE
+    DESCRIPTION "build bit-reproducible package"
+    DEFAULT ON
+    # Required for Debian, soon the world ...
+)
+
 # TODO Remove this after REQUIRED_LANGUAGES
 if( ENABLE_FORTRAN )
     # will set EC_HAVE_FORTRAN with the result
